internal/datastore/mysql/datastore_test.go

```diff
@@ -150,13 +150,12 @@ func GarbageCollectionTest(t *testing.T, ds datastore.Datastore) {
 	req.True(ok)
 
 	// Write basic namespaces.
-	_, err = ds.WriteNamespace(ctx, namespace.Namespace(
-		"resource",
-		namespace.Relation("reader", nil),
-	))
-	req.NoError(err)
-
-	writtenAt, err := ds.WriteNamespace(ctx, namespace.Namespace("user"))
+	writtenAt, err := ds.ReadWriteTx(ctx, func(ctx context.Context, rwt datastore.ReadWriteTransaction) error {
+		return rwt.WriteNamespaces(namespace.Namespace(
```

Comment:
Put a newline after `WriteNamespaces(` to make it more clear that you're writing two namespaces